ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -69,6 +70,9 @@ void py_module(py::module& module) {
     auto m_data_movement = module.def_submodule("data_movement", "data_movement operations");
     data_movement::py_module(m_data_movement);
 
+    auto m_downsample = module.def_submodule("downsample", "downsample operations");
```

Comment:
We don't need a dedicated submodule for downsample. Upsample and downsample ops should be in the same submodule

Reply:
probably best belongs to conv

Reply:
kept in data_movement as per discussion.

Reply:
Done.